layers/dependencies/nodejs/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "dependencies": {
-    "@aws-sdk/client-s3": "3.374",
-    "@aws-sdk/client-sns": "3.370",
-    "@aws-sdk/client-dynamodb": "3.370",
-    "@aws-sdk/client-lambda": "3.370",
-    "@aws-sdk/lib-dynamodb": "3.370",
-    "aws-sdk": "2.1420.0",
+    "@aws-sdk/client-s3": "3.421.0",
+    "@aws-sdk/client-sns": "3.421.0",
+    "@aws-sdk/client-dynamodb": "3.421.0",
+    "@aws-sdk/client-lambda": "3.421.0",
+    "@aws-sdk/lib-dynamodb": "3.421.0",
+    "aws-sdk": "2.1467.0",
```

Comment:
Does the Lambda layer still requires the older aws-sdk v2?

Reply:
No, it does not use aws-sdk v2. Thought I removed it but looks like I didn't. Thanks for catching that, I'll go ahead and remove it